docs/pages/visual-test/DateRange-compare.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import { Box, ColorSchemeProvider } from 'gestalt';
+import { DateRange } from 'gestalt-datepicker';
+
+export default function Snapshot() {
+  const [startDate, setStartDate] = useState<Date | null>(null);
```

Comment:
you need to run `yarn build` (stop the docs server first) and then `yarn playwright:update-visual-test playwright/visual-test/<filename>  --update-snapshots`